test/README.md

```diff
@@ -0,0 +1,60 @@
+# Testing
+
+## Requirements
+
+Make sure you have `opentofu` installed:
```

Comment:
I'd change this part to just terraform.
I wouldn't impose a strict requirement on either one. We generally test with terraform and if they diverge in the future this may be misleading.

Reply:
additionally in the docs we generally speak about `terraform` and `opentofu` may be confusing.
I'd stay with `terraform` for now to be consistent on a company level.
If you use `opentofu` locally and it works, no problem.

Reply:
I'm aware of the license change but I see it's very ambiguous on how it can be laid out.
Would you suggest changing the docs to opentofu as well?

I guess it's a general question we have to solve within the company whether we'll focus on Terraform or OpenTofu as they may diverge at some point. Same as we've observed with Elastic and OpenSearch.

Reply:
> Using Terraform with the new BSL license prevents us from embedding it in the product (source: [DuploCloud](https://duplocloud.com/blog/terraform-license-change-impacts-devops/)).

I think it would not be a problem, as we're not redistributing Terraform itself neither make profits of it by building a product around it.

Reply:
> I guess it's a general question we have to solve within the company whether we'll focus on Terraform or OpenTofu as they may diverge at some point. Same as we've observed with Elastic and OpenSearch.

Agree, if it's not a big effort, I'd suggest using usual Terraform for now.